### ISSUE TITLE
Custom text color for a specific tag

### ISSUE THREAD

**Issue by migfabio** [xhacker/TagListView#38]
Hi,

I've tried to customise color font for a specific tag. This is my code

```swift
let tagView: TagView = tagListView.addTag("Tag Test")
tagView.textColor = tag.fontColor
```
but I've received this error

**Value of type 'TagView' has no member 'textColor'**

Could you help me, please? Like make textColor public?

**Comment by xhacker** [comment]
I guess it’s because it’s not being declared as `public`: https://github.com/xhacker/TagListView/blob/master/TagListView/TagView.swift#L30-L34

I’m working on a fix.

**Comment by migfabio** [comment]
Thanks @xhacker 
It is what I said. It would be perfect if you will make public all properties. Just a suggestion.
Thank you for your work.

**Comment by xhacker** [comment]
Just realized it’s all fixed in #37. Please waiting for that to be merged. :)

**Comment by migfabio** [comment]
Brilliant! Thanks again

**Comment by xhacker** [comment]
Closed in b8aebdd36e9c465b8682a157bb16c218f17a013d.
